### ISSUE TITLE
Typo in context validation algorithm step from grammar adjustment needs fixing

### ISSUE THREAD

**Issue by dlongley** [w3c/vc-data-integrity#299]
A slight mixup here in the grammar fixes, this step should read "or if" not "then if".

_Originally posted by @dlongley in https://github.com/w3c/vc-data-integrity/pull/291#discussion_r1694302108_

**Comment by TallTed** [comment]
Given that the above change is correct, then the fix is somewhat larger.
```
If |contextValue| does not deeply equal |expectedContext|, then if any subtree in
|result|.|document| contains an `@context` property or any URI in
```
should become
```
If |contextValue| does not deeply equal |expectedContext|, any subtree in
|result|.|document| contains an `@context` property, or any URI in
```

**Comment by msporny** [comment]
Purely editorial, fixed in c887ab237b20b390d48871dcbb3e51fa4745f1e7, closing.
